packages/e2e/cypress/e2e/explore.cy.ts

```diff
@@ -304,4 +304,23 @@ describe('Explore', () => {
             });
         });
     });
+
+    it('Should clear query using hotkeys', () => {
+        cy.visit(`/projects/${SEED_PROJECT.project_uuid}/tables`);
+
+        cy.findByText('Orders').click();
+        cy.findByText('Is completed').click();
+
+        // run query
+        cy.get('button').contains('Run query').click();
+
+        // wait for query to finish
+        cy.findByText('Loading results').should('not.exist');
+
+        // clear query hotkeys
+        cy.get('body').type('{ctrl}{alt}{k}');
+
+        // verify empty selections
+        cy.findByText('Select a table').should('exist');
```

Comment:
We want to clear the explore page but not leave the selected table. The current `clear` action has a different purpose and should be renamed and we can add a new `clear` action for this use case.